includes/class-boldgrid-backup-archiver.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * File: class-boldgrid-backup-archiver.php
+ *
+ * @link https://www.boldgrid.com
+ * @since SINCEVERSION
+ *
+ * @package    Boldgrid_Backup
+ * @subpackage Boldgrid_Backup/includes
+ * @copyright  BoldGrid
+ * @version    $Id$
+ * @author     BoldGrid <support@boldgrid.com>
+ */
+
+/**
+ * Class: Boldgrid_Backup_Archiver
+ *
+ * Originally, all methods for archiving a file have lived in Boldgrid_Backup_Admin_Core. This class,
+ * over time, will absorb those methods.
+ *
```

Comment:
Not a change request, just a comment. I like the fact that you're moving methods out of the core, and into separate classes. We need more of this. the Admin core is waaaay to big.